### PR TITLE
Sprint 3 and first days of sprint 4

### PR DESCRIPTION
**Pull request by alopezna5**
Done tasks:

- https://trello.com/c/AOzIBD55/52-code-refactor-using-tabulate
- https://trello.com/c/e6Hsi1I1/54-improvements-fixes-and-omissions-in-scrpit
- https://trello.com/c/leT8HPsV/55-workaround-to-prevent-the-user-from-having-to-input-the-packagenameorigin
- https://trello.com/c/6lEhNLFQ/51-comments-and-docs-about-the-script
- https://trello.com/c/U3kid2y0/57-investigate-how-to-execute-the-script-without-using-python-or
- Proposed fixes from the previous PR

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,130 @@
+
+# Created by https://www.gitignore.io/api/python
+# Edit at https://www.gitignore.io/?templates=python
+
+### Python ###
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
```
Comment (davidmoremad) [inline .gitignore]: Faltaría añadir:
```
*.pyc
*Pipfile*
```

File: docs/build/html/_modules/index.html [hunk 2]
```diff
@@ -0,0 +1,194 @@
+
```
Comment (davidmoremad) [inline docs/build/html/_modules/index.html]: Los ficheros compilados (/build) no se suelen subir, estos son generados por ReadTheDocs, puedes meterlo también en el gitignore para facilitar esto.

File: .gitignore (hunk 1 above)
Comment (alopezna5) [reply .gitignore]: Done!

File: docs/build/html/_modules/index.html (hunk 2 above)
Comment (alopezna5) [reply docs/build/html/_modules/index.html]: Done!

**Pull request merged**